### ISSUE TITLE
Doesn't open on windows.

### ISSUE THREAD

**Issue by monoclex** [eul-im/eul#132]
video: https://upl.sirjosh3917.com/RepulsiveRockrat.mp4

(don't worry about the text font, that's when i customized windows)

i restarted my computer even and it still doesn't work, as well as running it with administrator rights

edit: using windows 8.1 industry embedded pro 64 bit

**Comment by spoltier** [comment]
I am seeing the same problem on windows 7. I suspect it may have to do with the antivirus interfering with volt setup in the background (but I can't be sure because I'm not able to disable it). Do you have anything of that kind (windows antimalware) running in the background ? Also, is it possible that windows embedded prevents writing to certain locations ?

**Comment by monoclex** [comment]
i don't think it prevents writing to specific locations, but i do have windows defeneder running so i'll disable that when i get time and see if that fixes it.

edit: apparently i have it turned off, but the Windows Defender Service still runs no matter what

**Comment by medvednikov** [comment]
Win defender shouldn’t be a problem.

I’ll release 0.81 with lots of fixes soon. Might fix yours.

**Comment by spoltier** [comment]
I just retested it with the new release on windows 7 - still happens (permanently asks to restart the app).
I have volt extracted in a folder in userspace, but I also tried running as admin 2x in a row and the behavior was the same.

**Comment by Didiv** [comment]
+1. same error on win7. volt.exe build 20.03.2019

**Comment by medvednikov** [comment]
This restart is required by the old terrible browser control. It needs to write to the registry to enable IE11 instead of IE6 (!)

Microsoft released a new webview control last year, I'm doing my best to migrate to it asap.

**Comment by spoltier** [comment]
I checked in the registry editor, HKEY_CURRENT_USER\SOFTWARE\Microsoft\Internet Explorer\Main\FeatureControl\FEATURE_BROWSER_EMULATION\volt.exe is set to 2af9, which seems to make sense.

I tried to set the other keys that could influence this, but it had no effect (see link below)

Note: I am doing this on Win7 _64 bit_.

See https://weblog.west-wind.com/posts/2011/May/21/Web-Browser-Control-Specifying-the-IE-Version#Registry-Key-Installation-for-your-Application

**Comment by gezar** [comment]
I have the same problem with Widows 10.
I hope it can be fixed soon...

**Comment by hideraimu** [comment]
Same issue on Win 7 x64

**Comment by Shagshag** [comment]
Same issue on Win 7 x64. Downloaded today.
